docs/Specifications.md

```diff
@@ -0,0 +1,75 @@
+# Functional Specifications
+
+Why do we write out our functional specifications? So no one says two days before the first match, "No one told me the robot had to be able to do *that*!"
+
+## Will Do / Can Do Table
+
+As we write our specification and code we will fill out this table.
+
+Category | Specification | Will Do | Chassis Can Do | Comp Can Do
+---------|---------------|---------|----------------|------------
+Move | Move responding to remote human manual control | YES | NO | NO
+Move | Translate and rotate at the same time | YES | NO | NO
+Shoot | Shoot cargo into high goal | YES | NO | NO
+Shoot | Shoot and Move | NO | NO | NO
```

Comment:
Why can't we do this? If we have a vison system that gives us the velocity of the target, or localization that gives us the velocity components of the robot. This is mathematically easy.

Reply:
Noted